### ISSUE TITLE
Calendar control : At 400% zooming Calendar Control after open is not visible. 

### ISSUE THREAD

**Issue by v-jashr** [microsoft/fluentui#12863]
Browser Details:​
Microsoft Edge 44.18362.449.0​
Chrome Version 1909(OS Build 18363.592)

Actual Result:​​
At 400% zooming Calendar Control after open is not visible.
Expected Result:​​
At 400% zooming Calendar Control after open is should be visible.

https://developer.microsoft.com/en-us/fluentui#/controls/web/calendar

able to reproduce on offical website of office fabric

![image](https://user-images.githubusercontent.com/50479173/80205551-6d6f4f00-8648-11ea-9913-8553c30f7650.png)

**Comment by aneeshack4** [comment]
@v-jashr thanks for reporting this to us. 400% zoom is a pretty specific scenario and seems uncommon but could be an accessibility use case @lorejoh12 can you please suggest a fix to this?

**Comment by ecraig12345** [comment]
We have an upcoming requirement to support 400% zoom (not sure of the timeline), so I'll leave this open and put it on the backlog.

**Comment by lorejoh12** [comment]
Copy/pasting from the other relevant bug, but I'll leave it here in case it's useful:

> Given that the component is prop driven, and there is a prop for having the day and month pickers overlap, I would say that it is the consumer's responsibility to attach a listener to the component for screen size changes/zoom and update the prop as necessary. I'd rather not add that functionality into the control itself because then the control is changing how it interprets the props it's given, rather than just doing what it's told, although I'm open to reviewing a PR for that functionality if someone would like to add it.
>
>The fact that the website isn't mobile friendly with the control means that we might consider adding something to the example page to fix it, but I don't know if we have that requirement yet, Elizabeth was talking about it in #12863 but I don't know if we have the 400% requirement yet.

**Comment by chce** [comment]
Which WCAG success criterion is not met here?

**Comment by MichaelRechenberg** [comment]
@joschect / @lorejoh12 When is this issue scheduled out to be resolved?  Will it, by chance, be fixed by 6/30/2021?

**Comment by dmaster7** [comment]
same question here - will this be fixed soon? 

> @joschect / @lorejoh12 When is this issue scheduled out to be resolved? Will it, by chance, be fixed by 6/30/2021?

**Comment by ecraig12345** [comment]
This is tracked internally [here](https://dev.azure.com/microsoftdesign/fluent-ui/_workitems/edit/11595) and should be resolved by 6/30.
